<launch>
    <arg name="low_th" default="0.5"/>
    <arg name="linear_vel" default="0.2"/>
    <arg name="angular_vel" default="0.349066"/>
    <arg name="topic_vel" default="/cmd_vel"/>
    <arg name="topic_voice" default="/Tablet/voice"/>
    <node pkg="teleop_voice" name="teleop_voice" type="teleop_voice.py" output="screen">
        <param name="low_th" value="$(arg low_th)" />
        <param name="linear_vel" value="$(arg linear_vel)" />
        <param name="angular_vel" value="$(arg angular_vel)" />
        <param name="topic_vel" value="$(arg topic_vel)" />
        <param name="topic_voice" value="$(arg topic_voice)" />
    </node>
</launch>